test/k8sT/Updates.go

```diff
@@ -11,6 +11,20 @@ import (
 )
 
 var _ = Describe("K8sUpdates", func() {
+
+	// This test runs 8 steps as following:
+	// 1 - delete all pods. Clean cilium, this can be, and should be achieved by
+	// `clean-cilium-state: "true"` option that we have in configmap
+	// 2 - install cilium `cilium:v1.1.4`
+	// 3 - make endpoints talk with each other with policy
+	// 4 - upgrade cilium to `k8s1:5000/cilium/cilium-dev:latest`
+	// 5 - make endpoints talk with each other with policy
+	// 6 - downgrade cilium to `cilium:v1.1.4`
+	// 7 - make endpoints talk with each other with policy
+	// 8 - delete all pods. Clean cilium, this can be, and should be achieved by
```

Comment:
By reading this, it makes it sound like as if we don't upgrade back to `k8s1:5000/cilium/cilium-dev:latest`?

Reply:
You're correct. I'm assuming all other tests will make sure if Cilium is up and running. Which the BeforeAll does that already. Should we upgrade back to `k8s1:5000`? Maybe we should. I'll add to the afterall

Reply:
Done

Reply:
Thanks!